### DISCUSSION TITLE
PYBv1.1 board not detected as a USB device on computer

### DISCUSSION THREAD

**Discussion by Sfrap99** [micropython/micropython#14271] [Hardware & Peripherals]
I am trying to use a new PYBv1.1 micropython board  (https://store.micropython.org/product/PYBv1.1) with a Pycontrol board (like this one: https://www.labmaker.org/products/pycontrol-breakout-board). I connected the micropython board to my computer via a micro-USB to USB connection and the Pycontrol board to a 12V power supply. When I do this, the red, blue and yellow lights light up and stay lit. 

With this setup my board is not detected by the computer (i.e., it doesn't pop up as a USB device). This means I can't really trooubleshoot by updating drivers/firmware because the computer just doesn't read the board. 

I have tried with different computers, USB cables and with 2 microcontrollers and Pycontrol boards but nothing works! I have also tried pressing the reset button but nothing changes.

Any ideas????

**Comment by robert-hh** [comment]
Does the PYBV1.1 still work without the labcontrol board? Did you install MicroPython to Pyboard?

**Comment by Sfrap99** [comment]
Hi Robert, thanks for your reply!! We get the same issue when the board is on its own. We didn't install Micropython to Pyboard, what is that? The only software we installed is the Pycontrol GUI.

**Reply by robert-hh** [reply]
MicroPython is the standard software for the Pyboard. It is a special version of Python. See here for the firmware https://micropython.org/download/PYBV11/, and here the project page: https://micropython.org/

**Reply by Sfrap99** [reply]
Thanks for the info, unfortunately we've already visited these pages. I believe we have downloaded the correct firmware, but we cannot flash/upload the firmware to the micropython as it is not visible on our computer. This is where we are stuck...

**Reply by robert-hh** [reply]
Did you follow the instructions on the download page? The usual too for downloading the firmware is dfu-util. If the MicroPython bootloader is not working, you have to connect the boot pin to 3v3 and push reset as documented in the instructions to set the device to bootload mode.

**Comment by Josverl** [comment]
Likely relevant as the PC needs to load drivers, 
What OS and CPU architecture is your host computer running?

Also, I'm working on a tool to make updating firmware simpler.
Its still early in the lifecycle, but it may work for you. No guarantee, but any feedback is appreciated if you want to give it a try..

https://pypi.org/project/mpflash/
